index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="main.css">
+    <script src="https://kit.fontawesome.com/e98a0f000a.js" crossorigin="anonymous"></script>
+    <title>About Page</title>
+</head>
+<body>
+        <header class="header">
+            <nav class="nav">
+                <div class="logodiv">
+                    <ul class="headercontent">
+                        <li> 
+                            <i class="language nav1-link fab fa-facebook-f"></i>
+                        </li>
+                        <li> 
+                            <i class="language nav1-link fab fa-twitter"></i>
+                        </li>
+                        <li><a class="lang">English</a></li>
+                        <li><a class="lang">My Page</a></li>
+                        <li><a class="lang">Logout</a></li>
```

Comment:
- [ ] Please make sure to follow the given design in all the aspects which are the size of the text. Also, kindly enhance the size of your logo as well. 🙏 
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/164484782-85df2207-7671-4c9a-8df4-a52ccdbddfb3.png)